moped-editor/src/auth/user.js

```diff
@@ -294,6 +295,8 @@ export const findHighestRole = roles => {
       return "moped-editor";
     case findRole("moped-viewer"):
       return "moped-viewer";
+    case findRole(nonLoginUserRole):
+      return nonLoginUserRole;
```

Comment:
These two lines and the import for `nonLoginUserRole` are the only changes - the rest are Prettier formatting ✨